packages/site/src/components/TritonSideDrawer.tsx

```diff
@@ -129,7 +142,19 @@ export function TritonSideDrawer() {
 
   return (
     <SideDrawer open={tritonOpen} onRequestClose={onCloseTriton}>
-      <SideDrawer.Toolbar>
```

Comment:
The Toolbar is meant to hold things like actions beneath the header, not the entire contents of the SideDrawer